worksheet_B/README.md

```diff
@@ -0,0 +1,60 @@
+INPUT random 5 letter word from given list i.e."Sheep", "Towel", "Light", "Learn", "Third", "Every"
+
+DECLARE word variable
+
+DECLARE correctanswer variable
+
+DECLARE guesscount variable
+
+SET correctanswer to False
+
+SET guesscount to 0
+
+WHILE guesscount is less than 4
+
+    DISPLAY 'You have a 5 letter word'
+
+    DISPLAY 'Can you guess what it is? Enter here:'
+
+	  User INPUT guess
+
+	  DECLARE guess variable
+
+	  IF guess is equal to 5
+		    ARRAY word variable, DECLARE as 'wordlist'
+		    ARRAY guess variable, DECLARE as 'guesslist'
+		    SET matching to 0
+
+        FOR i in range(0, 5):
+		  IF guesslist [i] is the same as wordlist [i]:
+			SET matching to matching + 1
+		IF matching is 0
+			DISPLAY 'Score: 0/5'
+			SET guesscount to guesscount + 1
+		IF matching is 1
```

Comment:
This code is repetitive and could have been replaced with a single if statement, e.g.
```
IF matching < 5
   DISPLAY 'Score: {matching}/5'
   SET guesscount to guesscount+1
```